app/utils/countryOptions/CountryOptions.scala

```diff
@@ -20,14 +20,22 @@ import com.typesafe.config.ConfigException
 import config.FrontendAppConfig
 import javax.inject.{Inject, Singleton}
 import play.api.Environment
+import play.api.i18n.Messages
 import play.api.libs.json.Json
 import utils.InputOption
 
 
 @Singleton
 class CountryOptions @Inject()(environment: Environment, config: FrontendAppConfig) {
 
-  def options: Seq[InputOption] = CountryOptions.getCountries(environment, config.locationCanonicalList)
+  def options()(implicit messages: Messages): Seq[InputOption] = {
+    CountryOptions.getCountries(environment, getFileName)
+  }
+
+  def getFileName()(implicit messages: Messages) = {
+    val isWelsh = messages.lang.code == "cy"
```

Comment:
`"cy"` is also referenced in the FrontendAppConfig class. It could be worth creating final values for some of these strings ("cy", "en", "GB" etc.)

eg.
https://github.com/hmrc/maintain-beneficiaries-frontend/blob/e6357c4e479306b4a359c42e3c07dfa065c50f16/app/config/FrontendAppConfig.scala#L31-L32
https://github.com/hmrc/maintain-beneficiaries-frontend/blob/e6357c4e479306b4a359c42e3c07dfa065c50f16/app/config/FrontendAppConfig.scala#L70-L73

Reply:
Updated